pkg/apis/clouddns/v1/integration_test.go

```diff
@@ -0,0 +1,6 @@
+//go:build integration
+// +build integration
+
+package v1
+
+const useMock = false
```

Comment:
This is set to true when we are running integration tests. I think that we are using mocks here is just a consequence. So i would probably rename it to `isIntegrationTest` or something similar, since some tests wont even execute when this variable is set

Reply:
Good point, changed :)